kickstarts/base.ks.erb

```diff
@@ -102,13 +102,17 @@ EOF
 # Let's rebuild the ramfs with with base scsi drivers we need
 kversion=$(rpm -q kernel --qf '%{version}-%{release}.%{arch}\n')
 ramfsfile="/boot/initramfs-$kversion.img"
+<% if @target == "vsphere" %>
```

Comment:
Can we add this to the existing `case @target` statement on line 109 rather than a separate `if ... else ... end`?

Reply:
https://github.com/ManageIQ/manageiq-appliance-build/pull/278/files#diff-97b85e7883f17153493a362c11fb3fe3L105 apply to all but vsphere, so I moved vsphere up outside of `case`, but I can change if the other way reads better.

Reply:
The multiple levels of indentation are hard to read here. I think I would rather have all the drivers listed out by target and just remove the initial assignment of `drivers="vmw_pvscsi mptspi"`. So something like:

```erb
<% case @target
   when "azure", "hyperv" %>
drivers="mptspi mptbase mptscsih hv_storvsc hid_hyperv hv_netvsc hv_vmbus"
<% when "ec2" %>
drivers="mptspi mptbase mptscsih xen-blkfront xen-netfront"
<% when "vsphere" %>
drivers="mptspi vmw_pvscsi"
<% end %>
/sbin/dracut --force --add-drivers "$drivers" $ramfsfile $kversion
```

What do you think?

Reply:
Moved `vsphere` to `case` statement.